keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -117,7 +117,7 @@ class MemoryCloudDriverCache(
           .map { it.availabilityZone }
           .toSet()
       }
-    }
+    }!!
```

Comment:
Caffeine is actually annotated with @Nullable on its interface - I went with `!!` instead of `?: emptySet()` because the loading function should always return a Set and it seems like an assertion type failure if it doesn't rather than silently running and returning emptySet

Reply:
That feels right.

Reply:
Yes. This is because sometimes absence is a valid response (e.g. user not found when looking up by email). It matches `Map.computeIfAbsent` syntax, so better match. Since you know it can never happen, this is perfect.